client/src/components/pages/Newpassword.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useContext, useEffect } from "react";
+
+import logo from "../assets/logo.svg";
+import authsvg from "../assets/loginsvg.svg";
+
+import "../css/login.css";
+
+import Box from "@mui/material/Box";
+import CircularProgress from "@mui/material/CircularProgress";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useNavigate,useParams } from "react-router-dom";
+
+//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
```

Comment:
Remove comment from here, seems like it contains sensitive details.

Reply:
sure